distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Expand_Objects_Helpers.enso

```diff
@@ -149,3 +149,8 @@ create_table_from_objects (base_value : Convertible_To_Rows) (fields : Vector |
                     if discovered_field_names.is_empty then Error.throw (Illegal_Argument.Error "Unable to generate column names as all inputs had no fields.") else
                         discovered_field_names.to_vector.map column_dict.get
             Table.new columns
+
+private fill_builder builder size:Integer =
```

Comment:
If it's used only inside of this module, shall we add an underscore to the name?